From 0e901e2bbe21ede55bc749c0084f4dae049bb952 Mon Sep 17 00:00:00 2001
From: Robert Lehmann <robert.lehmann@sitec-systems.de>
Date: Wed, 3 Feb 2016 08:56:54 +0100
Subject: [PATCH 15/15] boards.cfg: Add a S2 only configuration

This is needed to compile a S2 specific bootloader which defines some
things which are S2 dependent

Ticket #910
---
 boards.cfg | 1 +
 1 file changed, 1 insertion(+)

diff --git a/boards.cfg b/boards.cfg
index 0395d77..cf06b5a 100644
--- a/boards.cfg
+++ b/boards.cfg
@@ -368,6 +368,7 @@ Active  arm         armv7          mx6         freescale       mx6sx_19x19_arm2
 Active  arm         armv7          mx6         freescale       mx6sx_19x19_arm2  mx6sx_19x19_lpddr2_arm2_qspi2       mx6sx_19x19_arm2:IMX_CONFIG=board/freescale/mx6sx_19x19_arm2/imximage_lpddr2.cfg,MX6SX,LPDDR2,DEFAULT_FDT_FILE="imx6sx-19x19-arm2.dtb",SYS_BOOT_QSPI
 Active  arm         armv7          mx6         freescale       mx6sxscore       mx6sxscore                        mx6sxscore:IMX_CONFIG=board/freescale/mx6sxscore/imximage.cfg,MX6SX                                                               Robert Lehmann <robert.lehmann@sitec-systems.de>
 Active  arm         armv7          mx6         freescale       mx6sxscore       mx6sxscore_spi                        mx6sxscore:IMX_CONFIG=board/freescale/mx6sxscore/imximage.cfg,MX6SX,SYS_BOOT_SPINOR                                                               Robert Lehmann <robert.lehmann@sitec-systems.de>
+Active  arm         armv7          mx6         freescale       mx6sxscore       mx6sxscore_s2                        mx6sxscore:IMX_CONFIG=board/freescale/mx6sxscore/imximage.cfg,MX6SX,SYS_BOOT_SPINOR,SITEC_S2                                                               Robert Lehmann <robert.lehmann@sitec-systems.de>
 Active  arm         armv7          mx6         freescale       mx6sxsabresd     mx6sxsabresd                         mx6sxsabresd:IMX_CONFIG=board/freescale/mx6sxsabresd/imximage.cfg,MX6SX                                                               Fabio Estevam <fabio.estevam@freescale.com>
 Active  arm         armv7          mx6         freescale       mx6sxsabresd     mx6sxsabresd_qspi2                mx6sxsabresd:IMX_CONFIG=board/freescale/mx6sxsabresd/imximage.cfg,MX6SX,SYS_BOOT_QSPI                                                               Fabio Estevam <fabio.estevam@freescale.com>
 Active  arm         armv7          mx6         freescale       mx6sxsabresd     mx6sxsabresd_m4fastup          mx6sxsabresd:IMX_CONFIG=board/freescale/mx6sxsabresd/imximage.cfg,MX6SX,SYS_BOOT_QSPI,SYS_AUXCORE_FASTUP                            Fabio Estevam <fabio.estevam@freescale.com>
-- 
1.9.1

